main.tf

```diff
@@ -249,6 +249,45 @@ resource "azurerm_network_security_group" "private_agents" {
     destination_address_prefix = "*"
   }
 
+  security_rule {
+    name                       = "allowRDP"
+    description                = "Allow RDP access"
+    priority                   = 110
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "Tcp"
+    source_port_range          = "*"
+    destination_port_range     = "3389"
+    source_address_prefix      = "*"
+    destination_address_prefix = "*"
+  }
+
+  security_rule {
```

Comment:
we must not allow winrm calls via http, this is nothing we should support.
we should make sure everything is secured as much as possible.